sylvia/examples/generics.rs

```diff
@@ -9,9 +9,14 @@ where
     Msg: std::fmt::Debug + PartialEq + Clone + schemars::JsonSchema,
 {
     type Error;
+    type ExecC: CustomMsg;
```

Comment:
The idea of this test is that the interface is generic over `Msg` being basically `ExecC`.

Here it should be `#[sv::custom(msg=Msg)]`.

However it doesn't really help with this strange fail.

Reply:
yes, and as warning would show only on nightly it is for now pointless to merge it.
This PR will be merged once we figure out how to properly approach it